examples/experimental/dagster-blueprints/dagster_blueprints/cli.py

```diff
@@ -84,9 +142,68 @@ def load_blueprints_loaders_from_module_path(module_path: str) -> Sequence[YamlB
     return list(find_objects_in_module_of_types(module, YamlBlueprintsLoader))
 
 
+def has_vscode_cli_command() -> bool:
+    return bool(shutil.which("code"))
+
+
+def run_vscode_cli_command(args: List[str]) -> bytes:
+    return subprocess.check_output(["code"] + args)
+
+
+def recommend_yaml_extension() -> None:
+    if not has_vscode_cli_command():
+        click.echo(
+            "Could not find `code` executable in PATH. In order to use the dagster-blueprint-schema extension, "
+            "please install the redhat.vscode-yaml extension manually."
+        )
+        return
+
+    extensions = run_vscode_cli_command(["--list-extensions"]).decode("utf-8").split("\n")
+    if "redhat.vscode-yaml" in extensions:
+        click.echo("redhat.vscode-yaml extension is already installed.")
+    else:
+        if click.confirm(
+            "The redhat.vscode-yaml extension is not installed. Would you like to install it now?"
+        ):
+            run_vscode_cli_command(["--install-extension", "redhat.vscode-yaml"])
+
+
+def install_yaml_schema_extension(
+    dot_vscode_path: Path, yaml_dirs: List[Path], schema_paths: List[Path]
+) -> None:
+    """Builds a VS Code extension which provides the Red Hat YAML extension with the JSON schema files which
```

Comment:
Nitpick: I found this wording confusing on the first couple reads; thought it meant that we build a VS Code extension that wraps the Red Had YAML extension.